packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -34,19 +34,25 @@
         "
     >
         <textarea
+            wire:ignore
```

Comment:
We cannot `wire:ignore` here unfortunately since it would break dynamically making the textarea `disabled` for example

Reply:
hmmm. I don't know any other way to do it then, since the livewire request is resetting the `$initialHeight` in the php evertime it is rerendered.

Reply:
You could store the initial height in the value of a hidden input within the component, and then reference that in the JS instead of a property